docs-2.0/3.ngql-guide/10.tag-statements/3.alter-tag.md

```diff
@@ -0,0 +1,37 @@
+# ALTER TAG
+
+```ngql
+ALTER TAG <tag_name>
+    <alter_definition> [, alter_definition] ...]
+    [ttl_definition [, ttl_definition] ... ]
+
+alter_definition:
+| ADD    (prop_name data_type)
+| DROP   (prop_name)
+| CHANGE (prop_name data_type)
+
+ttl_definition:
+    TTL_DURATION = ttl_duration, TTL_COL = prop_name
+```
+
+`ALTER TAG` alters the structure of a tag with the given name in a graph space. You must have the `ALTER` privilege for the graph space. (TODO: privilege doc) To alter a tag in a specific graph space, you must use the graph space first.
```

Comment:
> You must have the `ALTER` privilege for the graph space. (TODO: privilege doc) To alter a tag in a specific graph space, you must use the graph space first.

These can be listed (unordered list) below a subtitle like `Prerequisites`.
The same with the other docs.

Reply:
If so, the `Prerequisites` only contains one sentence. I think it's not wise. Reference to mysql doc.